pkg/exec/helpers.go

```diff
@@ -86,6 +86,14 @@ func OutputLines(cmd Cmd) (lines []string, err error) {
 	return lines, err
 }
 
+// Output is like os/exec's cmd.Output, but over our Cmd interface
+func Output(cmd Cmd) ([]byte, error) {
```

Comment:
we don't need this if we use `docker inspect` to detect the network

Reply:
There's no reason not to have this, and docker inspect involves moving more data that we don't use.

Reply:
yeah, answered above, the current method is better